pepper-apis/src/main/java/org/broadinstitute/ddp/util/MiscUtil.java

```diff
@@ -62,6 +62,16 @@ public static void checkRegexPattern(String pattern) {
         }
     }
 
+    public static boolean isEmailFormatValid(String email) {
+        String emailRegex = "^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,7}$";
```

Comment:
This regex does not properly handle TLDs longer than 7 characters. Do we really need to perform strict checking here? Unless we're going to be a lot more stringent with the regexp (a full RFC 5222-compliant regex is an absolute beast), we might want to just do a check for a `@` followed by at least one `.`- it'll let a lot of invalid addresses, but we can rely on the SMTP server to catch the bad ones when it goes to send.

Reply:
How about this regex: "^[A-Za-z0-9+_.-]+@(.+)$"

Reply:
Per RFC 6532, email addresses were extended to support full UTF-8 characters in both the local part and the domain part so just checking for the english & numeric digits will kick out valid addresses as well. That regex also does not require at least one period in the domain-part- `test@domain` is accepted.

If we want to go dead simple, `\S+\@\S+\.\S+` is adequate.